### PR TITLE
Feat/add bioschemas support

### PR DESCRIPTION
**Pull request by svituz**
This PR adds Bioschemas support for collections and biobanks.
The implementation embeds json-ld schema to collections and biobanks cards.
The Biobanks have been mapped to DataCatalog and Dataset.
For the collections, to include some attribute not provided by the schema in Bioschemas.org (e.g., diagnosis available), the `additionalPropery` property has been used. In some cases, ontology terms are still required and will be included in the data model.

#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] Clean commits
- [ ] No warnings during install

### CODE REVIEW

File: src/components/cards/BiobankReportCard.vue [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="mg-biobank-card container">
+    <script v-html='jsonld' type='application/ld+json' />
```
Comment (jelmerveen) [inline src/components/cards/BiobankReportCard.vue:3]: I'm not sure what you are trying to achieve here? Can you elaborate

Comment (svituz) [reply src/components/cards/BiobankReportCard.vue:3]: Basically, the task is to add Bioschemas for Biobanks and Collections. This embeds the json-ld bioschemas representation of the biobank in BiobankReportCard. The schema is taken from the `jsonld` variable which is computed in  mapBiobankDataCatalog function

Comment (jelmerveen) [reply src/components/cards/BiobankReportCard.vue:3]: That I understand, but what is the purpose of having this embedded on the page within script tags, that is what i mean

Comment (svituz) [reply src/components/cards/BiobankReportCard.vue:3]: Bioschemas.org suggests using json-ld format and embedding it in the web page. https://bioschemas.org/tutorials/what_why_schema

Comment (jelmerveen) [reply src/components/cards/BiobankReportCard.vue:3]: thank you! For your explanation

File: src/utils/bioschemasMapper.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    }
+  })
+  return jsonld
+}
+
+export const mapBiobankDataCatalog = (biobank) => {
+  return {
+    '@context': 'https://schema.org',
+    '@type': 'DataCatalog',
+    '@id': `${getBaseUrl()}/biobank/${biobank.id}`, // TODO: Change with the persistent identifier
```
Comment (jelmerveen) [inline src/utils/bioschemasMapper.js:160]: maybe address this todo?

File: src/utils/bioschemasMapper.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
+import { getBaseUrl } from '@/utils/'
+
+// Should be better to put in the data model
```
Comment (jelmerveen) [inline src/utils/bioschemasMapper.js:3]: Can this be done for this PR?

File: src/utils/bioschemasMapper.js (hunk 2 above)
Comment (svituz) [reply src/utils/bioschemasMapper.js:160]: Currently, to my knowledge, the biobanks don't have a persistent universal identifier, but I understood that there is a task about this. This is a reminder to change this whenever the persistent uid will be available

File: src/utils/bioschemasMapper.js (hunk 3 above)
Comment (svituz) [reply src/utils/bioschemasMapper.js:3]: We should ask the state of this to the data team.

File: src/components/cards/BiobankReportCard.vue (hunk 1 above)
Comment (jelmerveen) [inline src/components/cards/BiobankReportCard.vue:3]: better use v-text (security), could you make the property more specific? e.g. collectionBioschema / biobankBioschema

File: src/views/CollectionReport.vue [hunk 4]
```diff
@@ -76,8 +77,8 @@ import ReportListRow from '@/components/report-components/ReportListRow'
 import ReportSubCollection from '@/components/report-components/ReportSubCollection'
 import CollectionReportInfoCard from '@/components/cards/CollectionReportInfoCard'
 import CollectionSelector from '@/components/buttons/CollectionSelector'
-
 import { mapDetailsTableContent, mapCollectionsData, collectionReportInformation } from '@/utils/templateMapper'
+import { mapCollectionsData as mapCollectionsToJsonLD } from '@/utils/bioschemasMapper'
```
Comment (jelmerveen) [inline src/views/CollectionReport.vue:81]: Make this consistent with the other import. Try to make propertynames more specific. then you do not need to do this

e.g:
getCollectionBioschemaData

File: src/utils/bioschemasMapper.js (hunk 3 above)
Comment (esthervanenckevort) [reply src/utils/bioschemasMapper.js:3]: At the moment this is only added for materials and data_categories. You can retrieve the data with a GET request to https://directory.bbmri-eric.eu/api/v2/eu_bbmri_eric_material_types and  https://directory.bbmri-eric.eu/api/v2/eu_bbmri_eric_data_types respectively.
I have created a story for our team to add the IRIs also for sex (you would get the values from  https://directory.bbmri-eric.eu/api/v2/eu_bbmri_eric_sex_types) and to add tags to the properties.

I would suggest that for now this mapping is okay, but we amend it when this is done.

**Review by jelmerveen — APPROVED** [review]
approved, with comment:

```
At the moment this is only added for materials and data_categories. You can retrieve the data with a GET request to https://directory.bbmri-eric.eu/api/v2/eu_bbmri_eric_material_types and https://directory.bbmri-eric.eu/api/v2/eu_bbmri_eric_data_types respectively.
I have created a story for our team to add the IRIs also for sex (you would get the values from https://directory.bbmri-eric.eu/api/v2/eu_bbmri_eric_sex_types) and to add tags to the properties.
```

**Pull request merged**